src/metricscollector/db/sqldb/sqldb_test.go

```diff
@@ -0,0 +1,419 @@
+package sqldb_test
+
+import (
+	"metricscollector/config"
+	. "metricscollector/db/sqldb"
+	"metricscollector/metrics"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/lib/pq"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"os"
+	"time"
+)
+
+var _ = Describe("Sqldb", func() {
+	var (
+		conf       *config.DbConfig
+		db         *SQLDB
+		logger     lager.Logger
+		err        error
+		metric     *metrics.Metric
+		mtrcs      []*metrics.Metric
+		start      int64
+		end        int64
+		before     int64
+		appId      string
+		metricName string
+		appIds     []string
+	)
+
+	BeforeEach(func() {
+		logger = lager.NewLogger("sqldb-test")
+		dbUrl := os.Getenv("DBURL")
+		conf = &config.DbConfig{MetricsDbUrl: dbUrl, PolicyDbUrl: dbUrl}
```

Comment:
We need to be wiping the db here otherwise we will have contamination across tests.